composer.json

```diff
@@ -30,11 +30,12 @@
     },
     "require": {
         "php": "^7.1",
-        "composer-plugin-api": "^1.0",
+        "composer-plugin-api": "^1.0 || ^2.0",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
-        "composer/composer": "^1.5.2",
+        "composer/composer": "^1.5.2 || ^2.0@dev",
+        "composer/semver": "^1.5.1 || ^2.0@dev",
```

Comment:
@boesing probably no longer needed as we have already stable [semver 2.0](https://github.com/composer/semver/releases/tag/2.0.0).